Arrowgene.Ddon.GameServer/Handler/ProfileGetCharacterProfileHandler.cs

```diff
@@ -50,7 +50,7 @@ public override void Handle(GameClient client, StructurePacket<C2SProfileGetChar
             ntc.EquipJobItemList = targetClient.Character.Equipment.getJobItemsAsCDataEquipJobItem(targetClient.Character.Job);
             ntc.HideHead = targetClient.Character.HideEquipHead;
             ntc.HideLantern = targetClient.Character.HideEquipLantern;
-            ntc.JewelryNum = targetClient.Character.JewelrySlotNum;
+            ntc.JewelryNum = (uint)(targetClient.Character.JewelrySlotNum + targetClient.Character.ExtendedParams.JewelrySlot);
```

Comment:
Why do this instead of simply updating JewelrySlotNum?

Reply:
This is a fair point. I was thinking about this while out shopping. Basically, we have 2 datapoints. There is a `jewelry_slot_num` inside the `ddon_character_common` table. Also, I created a new table called `ddon_orb_gain_extend_param` to match the values we obtained from learning the abilities. I think what maybe we should do is remove the ring number from `ddon_character_common`. Then when the character is loaded, apply the extended params to the character object. There we can always just assume the character has `1 + ExtendedParams.jewelry_slot`. Then I only need to update this character value in the menu after we buy from the dragon, otherwise it should be correct in all places used?

Reply:
Better have a named constant than a literal 1, and better have a method (or a getter?) to obtain the final jewelry slot count than do the operation in every place it's required, but yes, I think that's a good approach